spdx/config.py

```diff
@@ -12,22 +12,22 @@
 
 from __future__ import absolute_import
 from __future__ import print_function
+from __future__ import unicode_literals
 
-import csv
+import unicodecsv as csv
 import os
 
 from spdx.version import Version
 
 
 class TwoWayDict(dict):
 
-    def __setitem__(self, key, value):
-        dict.__setitem__(self, key, value)
-        dict.__setitem__(self, value, key)
-
-    def __delitem__(self, key):
-        dict.__delitem__(self, self[key])
-        dict.__delitem__(self, key)
+	def __setitem__(self, key, value):
+		dict.__setitem__(self, key,value)
+		dict.__setitem__(self, value,key)
+	def __delitem__(self, key):
```

Comment:
I may have missed it. I use tab for indentation which is set to default 4 spaces. Should I use 2 spaces ? I think the whole code is written with 4 spaces indentation.